src/createTokenBridgePrepareCustomFeeTokenApprovalTransactionRequest.ts

```diff
@@ -33,7 +34,7 @@ export async function createTokenBridgePrepareCustomFeeTokenApprovalTransactionR
     address: nativeToken,
     owner,
     spender: tokenBridgeCreatorAddressOverride ?? getTokenBridgeCreatorAddress(publicClient),
-    amount,
+    amount: amount ?? createTokenBridgeDefaultRetryablesFees,
```

Comment:
instead of approving max amount, we approve `0.02` eth which for custom gas token chains should be more than enough for retryables